dev/docker/hive/Dockerfile

```diff
@@ -83,15 +86,31 @@ RUN echo "HADOOP_CONF_DIR=${HADOOP_CONF_DIR}" >> /etc/environment
 RUN echo "HADOOP_CLASSPATH=${JAVA_HOME}/lib/tools.jar" >> /etc/environment
 RUN echo "YARN_HOME=${YARN_HOME}" >> /etc/environment
 RUN echo "HIVE_HOME=${HIVE_HOME}" >> /etc/environment
+RUN echo "ZK_HOME=${ZK_HOME}" >> /etc/environment
 RUN echo "PATH=${PATH}" >> /etc/environment
 RUN echo "CLASSPATH=${CLASSPATH}" >> /etc/environment
 RUN echo "LD_LIBRARY_PATH=${LD_LIBRARY_PATH}" >> /etc/environment
 
+################################################################################
+# install zoofkeeper
+ADD packages/zookeeper-${ZOOKEEPER_VERSION}.tar.gz /opt/
+RUN ln -s /opt/zookeeper-${ZOOKEEPER_VERSION} ${ZK_HOME}
+RUN mkdir $ZK_HOME/data
+
+################################################################################
+# install ranger hadoop plugin
+ADD packages/ranger-distro-${RANGER_VERSION}-hdfs-plugin.tar.gz /opt/
+RUN ln -s /opt/ranger-${RANGER_VERSION}-hdfs-plugin ${RANGER_HDFS_PLUGIN_HOME}
+
+################################################################################
+# install ranger hive plugin
+ADD packages/ranger-distro-${RANGER_VERSION}-hive-plugin.tar.gz /opt/
```

Comment:
I think `ranger-distro-${RANGER_VERSION}-hive-plugin.tar.gz` is incorrect, It should be `ranger-${RANGER_VERSION}-hive-plugin.tar.gz`?

Reply:
ok, fixed